libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -211,13 +211,19 @@ dictionary InvoicePaidDetails {
     string bolt11;
 };
 
+dictionary PaymentFailedData {
+    string error;
+    string node_id;
+    LNInvoice? bolt11;
```

Comment:
bolt11 is usually a string with the encoded payment request. Would in be clearer to rename the lnInvoice param to invoice?

Reply:
Agree. Done